deployment/scripts/webhook.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+export PATH=$PATH:/operator
```

Comment:
Why change the PATH? Isn't everything fully qualified below?

Reply:
ok. This is similar to the PATH settings used in `operator.sh` and I wasn't sure how the PATH is used. I'll go ahead and remove it from `webhook.sh`. I assume we can also similarly remove it from the `operator.sh`.

Reply:
Sounds good... I must have missed it in the other script. Either way, let's remove it if we can.